drivers/ads101x/ads101x.c

```diff
@@ -67,24 +63,35 @@ int ads101x_alert_init(ads101x_alert_t *dev,
     return _ads101x_init_test(I2C, ADDR);
 }
 
-int _ads101x_init_test(i2c_t i2c, uint8_t addr)
+static int _ads101x_init_test(i2c_t i2c, uint8_t addr)
 {
     uint8_t regs[2];
 
-    /* Acquire test */
     i2c_acquire(i2c);
-    if (i2c_init_master(i2c, I2C_SPEED) < 0) {
+
+    /* Register read test */
+    if (i2c_read_regs(i2c, addr, ADS101X_CONF_ADDR, &regs, 2, I2C_NOSTOP)
```

Comment:
I don't see any reason here to have a NOSTOP flag. (same at other place in the code)
It's not forbidden to read some regs send a STOP and then send data into another register.